lte/gateway/python/magma/enodebd/dp_client.py

```diff
@@ -45,3 +53,81 @@ def get_cbsd_state(request: CBSDRequest) -> CBSDStateResult:
         )
         return CBSDStateResult(radio_enabled=False)
     return res
+
+
+def _indoorTobool(s: str):
```

Comment:
this is redundant if you use a distutils' function called strtobool like this bool(strtobool(indoor_deployment))

Reply:
Almost redundant. Notice the value `indoor` which is something specific to Englewood BS.